### ISSUE TITLE
D needs a mini-unconf US based about 6 months before/after Dconf

### ISSUE THREAD

**Issue by truedat101** [truedat101/dlang-sv-community#52]
There are a lot of people who won't go to Europe for a conference, and there are some different formats of conference we might want to support in the US.

**Comment by truedat101** [comment]
There's an ugly and insightful thread on the dlang forums about this topic.  Apparently it's a religious issue.  I will try to summarize the points here, and then make a case for who cares, more conferences indicates a healthier community as opposed to bifurcation of energy.

**Comment by acehreli** [comment]
I've been following that thread. I agree partially: I think conferences should spend less time on talks and more time on activities like hackathon. The benefit would be two-fold: Attending the talks can be exhausting and discussing issues face-to-face is very valuable.

**Comment by truedat101** [comment]
But you should agree: there is a market for conferences that tailor to the technical content, governance, standards, and steering committee type stuff.  This conference may have wide appeal to core committers or groups with a lot vested in D, companies who are building on D, for example.

There is another market for conferences, networking and social interaction, which I personally enjoy.  There is definitely overlap between this market and the "org/technical" conference.

There is a different market for people who don't want the F2F interaction and would just like to sit in on a video stream.

And then there are the alternative formats:
- Hackathons
- Unconfs/Microconfs
- Trainings/Learning labs
- Maybe some other ideas not mentioned here

In any case, I was pro both sides of that thread, as I think the points are all valid on all sides.  And for that reason, I think people should be doing regionally tailored conference events that do not repeat material from D-conf and do not compete with resources or attention on DConf.

**Comment by pedroalvesbatista** [comment]
As long I'm more confident when working with D, I'm spreading it to my friends involved with backend and web development in general. I've applied to a call for paper in Campus Party, which is a tech event that happens in 12 countries, and in Brasil has about to 6 editions per year, so, is the only country where runs more than edition a year. I'll make a intro of D, and maybe a practical workshops on it, I would like some suggestions of what to teach, what to demonstrate.

The guys from Campus Party is open to partnerships, if someone from D community can come to some edition of Campus Party and show it's uses in industry or academia, that would help to spread the word around D. Maybe in the end of next year a mini D conf in America Latina could be held.

**Comment by truedat101** [comment]
Cool.  I would think the D community is interested in growing and getting some new people involved outside of Europe.

**Comment by truedat101** [comment]
Here's another perspective: http://subfurther.com/blog/2018/01/15/the-final-conf-down/

The iOS community went somewhat bonkers after 2012 when iOS really took off, and there were conferences like every week for something or another iOS dev related.  It's clear from reading this that the market for IOS dev confs has dried up a bit , although I don't know that this blogger has looked at the root causes with real objective viewpoints or even mentioned other reasons conferences might have a harder time.  A key point though is that there is (1) conference as academic publication and recognition (2) conference as entertainment/networking (3) conference as a place to learn (4) conference as a place to compete or participate AKA hackathon (5) conference as a place to market goods/services (6) conference as a business.  

I think the writer has pointed out very well that (6) the business of conferences has lost its viability.  And that may be fallout from the fact that items 2-5 have lost appeal.  The internet has largely replaced the (5) the need to market goods and services at a conference.  Youtube has made it possible to still attend conferences remotely, and in some ways diminishes the value of in person attendance (2) so things like TED talks become more valuable in their replay value on social media as opposed to the money to be made selling tickets to a ted talk.  

For D, we are really concerned with (1) to the degree that we consider discussing DIPS and projects is like academic publication, (2) networking, (3), and maybe (4).

**Comment by truedat101** [comment]
Here's a pretty nice microconf format, more on the technical side, but as an adjunct to an existing larger confrence: http://vger.kernel.org/lpc-bpf.html

A very focused subject matter, acceptance criteria is clear, and the speakers are all noted (with company) and format is clear.

**Comment by truedat101** [comment]
@jondegenhardt just mentioning you on this thread.  At some point we should adopt a format for what we want to do.

**Comment by pedroalvesbatista** [comment]
Agree, those formats enlights really well what is possible to lift off. Game industry is making huge use of D, running some experiments. I think we can start by these guys. Maybe some "web conference", where we set someone to present a subject using slides, maybe launch a hackathon over the globe so people perform coding, estabilish quick projects and some of those things. The field is green for possibilities !

**Comment by truedat101** [comment]
@acehreli and @jondegenhardt let's discuss tonight.
